docs/src/inflation/terminology.md

```diff
@@ -0,0 +1,53 @@
+
+
+### Inflation Rate [%]
+The Solana protocol will automatically create new tokens on a predetermined inflation schedule (discussed below). The *Inflation Rate [%]* is the annualized growth rate of the *Total Current Supply* at any point in time.
+
+
+### Inflation Schedule
+A deterministic description of token issuance over time. The Solana Foundation is proposing a dis-inflationary *Inflation Schedule*. I.e. Inflation starts at its highest value, the rate reduces over time until stabilizing at a predetermined long-term inflation rate (see discussion below). This schedule is completely and uniquely parameterized by three numbers:
+
+- **Initial Inflation Rate [%]**: The starting *Inflation Rate* for when inflation is first enabled. Token issuance rate can only decrease from this point. 
+- **Dis-inflation Rate [%]**: The rate at which the *Inflation Rate* is reduced.
+- **Long-term Inflation Rate [%]**: The stable, long-term *Inflation Rate* to be expected.
+
+
+### Effective Inflation Rate [%]
+
+The inflation rate actually observed on the Solana network after accounting for other factors that might decrease the *Total Current Supply*. Note that it is not possible for tokens to be created outside of what is described by the *Inflation Schedule*.  
+
+- While the *Inflation Schedule* determines how the protocol issues SOL, this neglects the concurrent elimination of tokens in the ecosystem due to various factors. The primary token burning mechanism is the burning of a portion of each transaction fee. While $100\%$ of each transaction fee is currently being destroyed, it is planned on reducing this burn rate to $50\%$ of each transaction fee, with the remaining fee to be retained by the validator that processes the transaction.
+- Additional factors such as loss of private keys and slashing events should also be considered in a holistic analysis of the *Effective Inflation Rate*. For example, it’s estimated that $10-20\%$ of all BTC have been lost and are unrecoverable and that networks may experience similar yearly losses at the rate of $1-2\%$.
+
+### Staking Yield [%]
+The rate of return (aka *interest*) earned on SOL staked on the network. It is often quoted as an annualized rate (e.g. "the network *staking yield* is currently $10\%$ per year").
+- *Staking yield* is of great interest to validators and token-holders holders who wish to delegate their tokens to avoid token dilution due to inflation (the extent of which is discussed below).
+- Inflationary issuances are expected to be split in distribution with 95% to be distributed to staked token-holders in proportion to the SOL they have staked with validators and 5% to be allocated to the Foundation treasury for operational expenses and future grants.
```

Comment:
Is this true?  I thought we are currently set with 100% to token-holders

Reply:
You are correct! I fixed it, thanks for the catch